.pre-commit-config.yaml

```diff
@@ -22,8 +25,16 @@ repos:
         exclude: ^api/scpca_portal/wsgi.py$
 
   - repo: https://github.com/psf/black
-    rev: 19.10b0
+    rev: 22.3.0
     hooks:
       - id: black
-        args: [--line-length=100]
         language_version: python3
+
+# re-enable after resolving eslint import error
```

Comment:
Not sure if you still need the commented code

Reply:
good catch we wont need this